app/models/organization.rb

```diff
@@ -0,0 +1,8 @@
+class Organization < ApplicationRecord
+  has_one_attached :file
```

Comment:
Can we name this `logo` so it's more clear what kind of file it is?

Reply:
@littleforest I renamed file to logo, but is there anything else I need to add? How does `Organization` know what `logo` is referring to?

Reply:
It doesn't really matter what the name is, it just makes for readability in the code. So you might have something like:  `if organization.logo.attached?`. It is just a little more clear what type of attachment we are expecting.